src/main/java/cloud/fogbow/ras/core/ApplicationFacade.java

```diff
@@ -462,6 +520,18 @@ protected void checkConsistencyOfEmbeddedOrder(Order mainOrder, Order embeddedOr
         return networkOrders;
     }
 
+    private void startOperation() {
+        while (SynchronizationManager.getInstance().isReloading())
```

Comment:
I wonder if it is possible to implement this without busy wait. Cannot think of any good solution using semaphores though.